### DISCUSSION TITLE
始终说找不到weight.pt

### DISCUSSION THREAD

**Discussion by AaronSongbin** [magicpdf/Magic-PDF#316] [General]
FileNotFoundError: [Errno 2] No such file or directory: 'models/MFD/weights.pt'

**Comment by AaronSongbin** [comment]
库中也没有这个文件啊
<img width="858" alt="image" src="https://github.com/user-attachments/assets/01e4120a-5a78-4e31-b761-d67b83a0d5bc">

**Reply by myhloli** [reply]
please read readme.md

**Reply by AaronSongbin** [reply]
感谢回复，我是新手，操作一步多要问一下gpt。还请见谅。

**Comment by YhZhang-code** [comment]
我也一直是这个问题，请问楼主解决了吗？
